docs/release-notes/rockcraft-1-7-0.rst

```diff
@@ -0,0 +1,106 @@
+.. _release-1.7.0:
+
+Rockcraft 1.7.0 release notes
+=============================
+
+20 December 2024
+
+Learn about the new features, changes, and fixes introduced in Rockcraft 1.7.0.
+For information about the Rockcraft release cycle, see the
+:ref:`release_policy_and_schedule`.
+
+
+What's new
+----------
+
+Rockcraft 1.7.0 brings the following features, integrations, and improvements.
+
+
+Support for remote builds
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We added support for remote builds, which leverage off-site compute for
+building rocks and can build for multiple architectures with a single command.
+Remote builds are hosted on Launchpad.
+
+This feature is experimental and subject to change, and more information is
+available by running ``rockcraft remote-build --help``. Feel free to provide
+feedback on the `Ubuntu Discourse`_ forum.
+
+Minor features
+--------------
+
+Rockcraft 1.7.0 brings the following minor changes.
+
+
+``init`` command
+~~~~~~~~~~~~~~~~
+
+The default base of the project file generated by ``rockcraft init`` has been
+updated from "ubuntu@22.04" to "ubuntu@24.04".
+
+
+Rockcraft snap
+~~~~~~~~~~~~~~
+
+The base of the Rockcraft snap was updated from core22 to core24.
+
+
+Go extension
+~~~~~~~~~~~~
+
+A new :ref:`tutorial <build-a-rock-for-a-go-application>` and :ref:`reference
+<go-framework-reference>` are available for the go-framework extension.
+
+
+OCI image configuration
+~~~~~~~~~~~~~~~~~~~~~~~
+
+When using ``run-user: _daemon_``, the value of the ``Config.User`` key in the
+generated rock has been updated from the user name (``_daemon_``) to its
+numeric ID.
+
+
+12-factor documentation
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The documentation for the various 12-factor framework extensions has been
+updated for clarity and to better conform with documentation best practices.
+
+
+Maven and Ant plugins
+~~~~~~~~~~~~~~~~~~~~~
+
+Changed the location of the Java runtime executable on rocks built using the
+Maven and Ant plugins from ``/bin/java`` to ``/usr/bin/java``, as the former
+conflicts with the user-merged Ubuntu base.
```

Comment:
I'm still not a fan of starting a sentence with a past tense verb, so let's insert ourselves:

```suggestion
We changed the location of the Java runtime executable on rocks built using the
the Maven and Ant plugins from ``/bin/java`` to ``/usr/bin/java``, as the
former conflicts with the user-merged Ubuntu base.
```

Reply:
I accepted this suggestion "manually", as I worked on the page here